admin/app/views/abtests.scala.html

```diff
@@ -39,7 +39,7 @@ <h3 class="abtests-expired-title">Expired tests <a href="#">show</a></h3>
 
     <script type="text/javascript">
         window.abSwitches = {
-            @conf.switches.Switches.all.filter(_.group == "A/B Tests").map{ s =>
+            @conf.switches.Switches.all.filter(_.group.name == "A/B Tests").map{ s =>
                 "@CamelCase.fromHyphenated(s.name)": @s.isSwitchedOn,
```

Comment:
You could replace `group.name == "A/B Tests"` with `group == SwitchGroup.ABTests`

That way, if the name changes (or the group is deleted) it should flag as a compilation error.